Mono.Cecil.Cil/MethodBody.cs

```diff
@@ -53,23 +53,38 @@ public sealed class MethodBody {
 		}
 
 		public Collection<Instruction> Instructions {
-			get { return instructions ?? (instructions = new InstructionCollection (method)); }
+			get {
+				if (instructions == null)
+					Interlocked.CompareExchange (ref instructions, new InstructionCollection(method), null);
```

Comment:
Nit pick: please add a space before the opening parentheses in `new InstructionCollection (` to match the overall codestyle.

Reply:
updated